css-animations-2/Overview.bs

```diff
@@ -847,6 +873,271 @@ expressed in milliseconds, it must be divided by 1,000 to produce a value in
 seconds before being assigned to the {{AnimationEvent/elapsedTime}} member of
 the {{AnimationEvent}}.
 
+# Animation Triggers # {#triggers}
+
+An <dfn export>animation trigger</dfn> is used to control the playback
+of its associated [=animation=] for time-driven animations.
+[=Animation triggers=] follow the same [[web-animations-1#hierarchical|hierarchy]]
+as animations and are associated with a [=timeline=] and an [=animation trigger effect|effect=].
+
+Issue: Should there be any effect of triggers on scroll-driven animations?
```

Comment:
I suspect triggers on scroll-driven animations should "just work". E.g. they could play / stop the animation when the trigger condition occurs / finishes.

Reply:
How does each `type` affects a scroll-driven animation?
Furthermore, what could be the actual use-case for this? Perhaps one that I can think of now is if we have a `hover-timeline` in the future and we want to enable/disable it on different scroll positions. Is that what you mean?